tests/ui/intern-local.js

```diff
@@ -15,5 +15,8 @@ define(['./_base', './_cli', 'intern'], function(config, cli, intern) {
         port: 4444
     };
 
+    // Format for outputting test results
+    config.reporters = ['Pretty'];
```

Comment:
We shouldn't use "Pretty" locally.  It squashes `console.log` messages and makes debugging tests difficult.